### ISSUE TITLE
Metricbeat system/process fails on process names containing parentheses 

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#5692]
Processes containing parentheses break the `/proc/[PID]/stat` parser used on Linux and FreeBSD.

As reported by @immon in https://github.com/elastic/gosigar/issues/81 an error is logged.

```
2017/10/30 16:05:31.762957 process_summary.go:79: ERR Unknown state <41> for process with pid 1467
```

For confirmed bugs, please report:
- Version: All
- Operating System: Linux / FreeBSD
- Steps to Reproduce: 

I have opened a PR at https://github.com/elastic/gosigar/pull/86

**Comment by ruflin** [comment]
As https://github.com/elastic/gosigar/pull/86 is now merged, we should probably make a gosigar release and update it in beats?

**Comment by andrewkroh** [comment]
Fixed in v6.1.0 by #5775.
